### PR TITLE
Set Up "Community Specification 1.0"

### PR DESCRIPTION
**Pull request by expede**
This PR adds a template for CLA and general IP release hygiene. This both makes it possible for others to build on top of UCAN, and prepares us for eventually housing UCAN inside of a spec org (e.g. IETF, W3C, DIF).

We need to go back and have existing contributors sign docs contained in this PR. It's best to do this early in the life of the project.

### CODE REVIEW

**Comment by expede** [comment]
Also FYI @bmann

**Comment by expede** [comment]
cc @csuwildcat -- The long awaited IPRing is here! Please let me know what you think.

Of course after merging this we will need to track down the handful of existing contributors, but wanted to run this past you first. The CommunitySpecification 1.0 was recommended by some folks inside the DIF.

**Comment by expede** [comment]
cc FYI @dholms, @matheus23, @Gozala, @b5

File: CONTRIBUTING.md
Comment (matheus23) [inline CONTRIBUTING.md:13]: Uh, this point reads weirdly in a spec repo, although generally NBD I think.

**Review by matheus23 — APPROVED** [review]

File: CONTRIBUTING.md
Comment (expede) [reply CONTRIBUTING.md:11]: Correct!

File: CONTRIBUTING.md
Comment (expede) [reply CONTRIBUTING.md:13]: Yeah. This is the spec standards verbiage directly 🤷‍♀️ I'll update for our use case 👍

Comment (expede) [reply CONTRIBUTING.md]: I just... removed that line

File: CODE_OF_CONDUCT.md
Comment (b5) [inline CODE_OF_CONDUCT.md:63]: Need a contact method here. Should also be clear _who_ will receive these reports. This is one of those examples of "organizational overhead" I'm known to harp about. I'd be happy to volunteer my name here if others deem appropriate, but would prefer at least two folks receive initial CoC-related reports.

Maybe we could even get fancy & align CODEOWNERS.md to folks who would receive initial CoC reports.

Delighted to see a CoC pulled from a reputable source.

Comment (expede) [reply CODE_OF_CONDUCT.md:63]: >  This is one of those examples of "organizational overhead" I'm known to harp about

Harp away — it's appreciated!

File: CODE_OF_CONDUCT.md
Comment (expede) [inline CODE_OF_CONDUCT.md:63]: @b5 better?

File: CODEOWNERS.md
Comment (expede) [inline CODEOWNERS.md:5]: @b5 great suggestion

File: CODE_OF_CONDUCT.md
Comment (bmann) [reply CODE_OF_CONDUCT.md]: We have the `ucan.xyz` domain so I'd propose we do some group email forwarding there. Hmm. Thinking about this some more, I actually do have some thoughts on how to do some shared inboxes. We may want an invite only "operations" repo to keep everything in one place.

Comment (expede) [reply CODE_OF_CONDUCT.md]: Great idea! It could even be public... it's more "uninteresting to others" than secret for this spec, and we want to keep as much open as possible in this case

**Review by dholms — APPROVED** [review]
🙌 🙌

File: CODE_OF_CONDUCT.md
Comment (b5) [reply CODE_OF_CONDUCT.md:63]: looks good to me!

**Review by b5 — APPROVED** [review]
👍

**Review by Gozala — APPROVED** [review]

**Comment by expede** [comment]
Yay, we did it!

**Pull request merged**
